server.go

```diff
@@ -68,8 +68,9 @@ type Server struct {
 	EventWorker *EventWorker
 	TraceWorker *TraceWorker
 
-	Statsd *statsd.Client
-	Sentry *raven.Client
+	Statsd   *statsd.Client
```

Comment:
(Well, we're not changing all the metrics yet, I guess. But eventually, this can go away)